README.md

```diff
@@ -101,6 +101,8 @@ For the versions available, see the [tags on this repository](https://github.com
 
 - Documentation : https://klaw-project.io/docs
 
+- Project documentation for contributors: [`./docs`](./docs)
```

Comment:
I think it would be great to have this as a separate headline with links to this doc as well as the docs in coral. That way it would be clear that

- `/docs` (in root of Klaw) is for technical documentation related to ALL Klaw and relevant for all contributers
- `coral/docs` is technical documentation related to the react app 

And in future there could also be additional docs, e.g. for `/core` etc.  

(We also could have the README in `/docs` have a section with a more detailed linking of different docs 🤔  that maybe be even better?)

Reply:
I think this is a good idea ^^ Do you want to push a commit for it? I don't want to get the vision wrong :o

Reply:
I think this is what you had in mind? :o [3869034](https://github.com/aiven/klaw/pull/884/commits/38690344c578149eb7a181da3ec76ccf4ba611d9)